pkg/utils/configmap.go

```diff
@@ -58,8 +58,8 @@ func getMetadata(route *routev1.Route) string {
 	return strings.TrimSpace(fmt.Sprintf(stubMetadata, host, host, host))
 }
 
-func getMetadataConfigMap(route *routev1.Route) *corev1.ConfigMap {
-	meta := defaultMeta()
+func GetMetadataConfigMap(route *routev1.Route) *corev1.ConfigMap {
```

Comment:
how is this something generic? Looks very specific for a usecase and should be near to the consumer.

Reply:
Refactored the function names to better reflect what they are doing now